src/main/java/group1/mips_simulator/components/memParts/CacheLine.java

```diff
@@ -0,0 +1,65 @@
+package group1.mips_simulator.components.memParts;
+
+import group1.mips_simulator.Utility;
+import group1.mips_simulator.components.Word;
+import group1.mips_simulator.components.Config;
+import group1.mips_simulator.components.memParts.Storage;
+
+
+public class CacheLine {
+    // one line has a block of 8 words;
```

Comment:
What's the logical difference between a line and a block? I thought they were two words for the same thing?